wren-ai-service/src/pipelines/generation/intent_classification.py

```diff
@@ -235,6 +238,7 @@ def prompt(
         query=query,
         db_schemas=construct_db_schemas,
         query_history=previous_query_summaries,
+        current_time=show_current_time(configuration.timezone),
```

Comment:
nitpick: the timezone in configuration class had a function to convert to current time, we might use it instead of this one.